datalake/backend/tests/utils.py

```diff
@@ -239,6 +244,45 @@ def __exit__(
         self._item.delete()
 
 
+class ProcessingAsset:
+    def __init__(
+        self,
+        asset_id: Optional[str] = None,
+        item_index: Optional[str] = None,
+        multihash: Optional[str] = None,
+        url: Optional[str] = None,
```

Comment:
All of the calls to this method include a `url` parameter, so it shouldn't be optional.